src/browser/api_protocol/api_handlers/application.js

```diff
@@ -336,7 +337,10 @@ function runApplication(identity, message, ack, nack) {
     const appIdentity = apiProtocolBase.getTargetApplicationIdentity(payload);
     const { uuid } = appIdentity;
     let remoteSubscriptionUnSubscribe;
-    let unsub = () => null;
+    const logMsg = manifestUrl ?
+        `unsub called before duplicate uuid transport listener removed for ${uuid}` :
+        '';
+    let unsub = () => writeToLog('error', logMsg);
```

Comment:
:neckbeard: `const`

Reply:
that will get reassigned further down

Reply:
Yup i'm blind https://github.com/HadoukenIO/core/pull/736/files#diff-dd883de5703030a371af2cdd9ad72616R376